dnf-testing.py

```diff
@@ -0,0 +1,335 @@
+
```

Comment:
I wouldn't mind having a shebang here. Is python 2 vs python 3 a problem on fedora, can't use a single shebang?

I don't like wrappers around wrappers...

Also, how about naming this better? It runs the tests in a container... I can't think of anything short and punchy, `run-tests-in-container` isn't that great :slightly_smiling_face:

Reply:
Not only on Fedora. The script is run in various environments - Fedora, RHEL7, RHEL8 with different versions of python installed. That is why single shebang is not usable and one of the reasons the shell wrapper exists.

Reply:
But I will think about some cooler name for the script.